internal/config/json_reporter.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+import "errors"
+
+type JSONReporterSettings struct {
+	Path string `hcl:"path" json:"path"`
+}
+
+func (ag JSONReporterSettings) validate() error {
```

Comment:
Why is the variable named ag? 🤔

Reply:
I thought you had to use obscure variable naming in Go? :p
Copy/Paste coding, fixed.